ezyhttp-client/src/main/java/com/tvd12/ezyhttp/client/request/Request.java

```diff
@@ -4,7 +4,7 @@
 
 import com.tvd12.ezyhttp.core.constant.HttpMethod;
 
-public interface Request {
+public interface Request<T> {
```

Comment:
In fact, in a get request, there is no request quest body, and you're rarely care about request body type, so you should not use generics type here